File: src/main/java/ru/translater/translation/yandex/model/TranslationYandexRequest.java
package ru.translater.translation.yandex.model;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

@Data
public class TranslationYandexRequest {
    private String sourceLanguageCode;
    private String targetLanguageCode;
    private String format;
    private String folderId;
    private List<String> texts = new ArrayList<>();
}

File: src/main/kotlin/com/translater/translate/yandex/model/TranslationYandexRequest.kt
package com.translater.translate.yandex.model

data class TranslationYandexRequest(
    var sourceLanguageCode: String? = null,
    var targetLanguageCode: String? = null,
    var format: String? = null,
    var folderId: String? = null,
    var texts: MutableList<String> = mutableListOf()
)